<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecalWin</class>
 <widget class="QDialog" name="RecalWin">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recalibration Parameters</string>
  </property>
  <widget class="QPushButton" name="timeRecalButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>132</y>
     <width>145</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Recalibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mzRecalButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>164</y>
     <width>145</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>m/z Recalibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>196</y>
     <width>145</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>36</y>
     <width>177</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>retention time window</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTimeWindow">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>32</y>
     <width>154</width>
     <height>27</height>
    </rect>
   </property>
   <property name="suffix">
    <string> seconds</string>
   </property>
   <property name="prefix">
    <string>+/- </string>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>120</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>68</y>
     <width>109</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>m/z window</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinMzWindow">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>64</y>
     <width>125</width>
     <height>27</height>
    </rect>
   </property>
   <property name="prefix">
    <string>+/- </string>
   </property>
   <property name="suffix">
    <string> Da</string>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
